docs/node/workflows.md

```diff
@@ -127,26 +154,18 @@ Operations like timers and Activites are cancelled by the cancellation scope the
 <!--SNIPSTART nodejs-shield-awaited-in-root-scope-->
 <!--SNIPEND-->
 
-### Callbacks and cancellation scopes
-
-Callbacks are not particularly useful in Workflows because all meaningful asynchronous operations return Promises.
-
-In the odd case that user code utilizes callbacks, CancellationScope.cancelRequested can be used to subscribe to cancellation.
-
-<!--SNIPSTART nodejs-cancellation-scopes-with-callbacks-->
-<!--SNIPEND-->
-
-## Scheduling Cron Workflows
+## Scheduling Cron Workflows (stub)
 
 There is no official support for Cron Workflows in Node.js yet.
 
-## Executing External Workflows
+## Executing External Workflows (stub)
```

Comment:
Hopefully we'll have those this week (child WFs + external WFs + signal external WF).